src/components/cdPipeline/CDPipeline.tsx

```diff
@@ -1396,6 +1491,19 @@ export default class CDPipeline extends Component<CDPipelineProps, CDPipelineSta
                 {this.state.pipelineConfig.isVirtualEnvironment
                     ? VirtualEnvSelectionInfoBar && <VirtualEnvSelectionInfoBar />
                     : this.renderTriggerType()}
+                {this.state.pipelineConfig.isVirtualEnvironment
```

Comment:
this should be `{this.state.pipelineConfig.isVirtualEnvironment && HelmManifestPush ?  <HelmManifestPush`